autoload/ensime.vim.py

```diff
@@ -280,11 +279,11 @@ def autocmd_handler(self, filename):
         self.vim.current.line = (
             'Autocmd: Called %s times, file: %s' % (self.calls, filename))
     # @neovim.function('EnCompleteFunc', sync=True)
-    def complete_func(self, args):
-        if args[0] == '1':
+    def complete_func(self, findstart, base):
+        if findstart == '1':
             self.complete()
             line = self.vim.eval("getline('.')")
-            start = self.cursor()[1] - 1
+            start = self.cursor()[1]
             pattern = re.compile('\a')
```

Comment:
This works by accident only, as there are almost never matching against `chr(0x07)` happens.
It should be something like `r'\b'`, like in #57 (90eb22874fcdf9f6f3e514068a90bc012cc91e83).